server/service/client_orbit.go

```diff
@@ -0,0 +1,77 @@
+package service
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type OrbitClient struct {
+	*baseClient
+	enrollSecret string
+	hardwareUUID string
```

Comment:
do we only use these two to enroll? if yes, it makes more sense to wend `enrollSecret` and `hardwareUUID` as parameters to `DoEnroll` instead

Reply:
in the previous implementation, it was doing exactly that..but it makes sense to have it here now that we have `OrbitClient` (https://github.com/fleetdm/fleet/pull/7246#discussion_r964959193)